rc/target_test.go

```diff
@@ -100,9 +100,18 @@ AA9WjQKZ7aKQRUzkuxCkPfAyAw7xzvjoyVGM5mKf5p/AfbdynMk2OmufTqj/ZA1k
 				})))
 				base, ok := (*transport).Base.(*http.Transport)
 				Expect(ok).To(BeTrue())
+
+				var expectedCaCertPool *x509.CertPool
+				if runtime.GOOS != "windows" {
+					expectedCaCertPool, err = x509.SystemCertPool()
+					Expect(err).NotTo(HaveOccurred())
+				} else {
+					expectedCaCertPool = x509.NewCertPool()
+				}
+
 				Expect((*base).TLSClientConfig).To(Equal(&tls.Config{
 					InsecureSkipVerify: true,
-					RootCAs:            nil,
+					RootCAs:            expectedCaCertPool,
```

Comment:
As far as I can tell, this should be equivalent. The Go docs for `RootCAs` say that if it's specified as `nil`, it uses the system cert pool. Were you able to confirm this fixes it?

Reply:
@vito after test locally I can confirm this PR wont fix the problem. It turned out that [x509/root_darwin.go](https://golang.org/src/crypto/x509/root_darwin.go) doesn't pick up *ANY* intermediate certificate from `/Library/Keychains/System.keychain`. In my case it doesn't pick up my `localhost intermediate ca`, not `Pivotal Intermediate CA` and not `DigiCert SHA2 ...` etc.  Note root CA is in yellow and intermediate ca is in blue.

<img width="580" alt="screen shot 2018-07-21 at 12 55 47 am" src="https://user-images.githubusercontent.com/1585949/43032273-0cc5465e-8c81-11e8-9edd-107cde35c2ac.png">

Not sure if we gonna jump in this rabbit hole but chaining the certs when config atc `tls_cert` is probably the best solutions for now. I could update the original issue and close this PR.